form/tests/__init__.py

```diff
@@ -0,0 +1,31 @@
+import json
+from unittest import mock, TestCase
+
+
+VALID_REQUEST_DATA = {
+    'data': '{"contact_name": "Test", "marketing_source_bank": "", '
+    '"website": ''"example.com", "exporting": "False", "phone_number": "",'
+    ' ''"marketing_source": "Social media", "opt_in": true, ''"marketing_s'
+    'ource_other": "", "email_address1": ''"test@example.com", "agree_term'
+    's": true, "company_name": "Example ''Limited", "email_address2": "tes'
+    't@example.com"}'
+}
+VALID_REQUEST_DATA_JSON = json.dumps(VALID_REQUEST_DATA)
+
+
+class MockBoto(TestCase):
```

Comment:
not a fan of fixtures, they tend to be scattered everywhere in the code base and you end up not knowing where they are defined!